src/assets/svg/kyndryl-logo.svg

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="125.74" height="40" version="1.1" viewBox="0 0 125.74 40" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Size doesn't match Figma (80px). The "Kyndryl" in both this and the bridge logo should be the same size it appears.

Reply:
Updated svg